sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/ParDo.java

```diff
@@ -224,18 +224,18 @@
  *             public void processElement(@Element String word, MultiOutputReceiver r) {
```

Comment:
:point_up:  There's an extra `}` generated on line 222 in the Javadoc...

I didn't find a way to remove it while using the `<pre>@code{` style. :/